btcpy/constants.py

```diff
@@ -0,0 +1,58 @@
+from btcpy.lib.codecs import Base58Codec, Bech32Codec
```

Comment:
This looks like a circular import (this imports `codecs`, `codecs` imports `setup`, `setup` imports `constants`). Please double-check with unit tests.

Reply:
if it would be unittest would fail on unit.py:26 but the unit test result is same as on master